doc/src/modules/index.rst

```diff
@@ -5,9 +5,12 @@ SymPy Modules Reference
 
 .. module:: sympy
 
-Because every feature of SymPy must have a test case, when you are not sure how
-to use something, just look into the ``tests/`` directories, find that feature
-and read the tests for it, that will tell you everything you need to know.
+This section gives the detailed documentations for each module in SymPy.
+
+Every feature of SymPy have a test case. When you are not sure how to use
+something, just look into the example section in docstrings or ``tests/``
+directories. Find the feature and read the tests for it. It will tell you
+everything you need to know.
```

Comment:
IMHO we shouldn't really be recommending people to use the tests as documentation. We should have proper examples and real docstrings.

Reply:
Fixed it.